hive/pom.xml

```diff
@@ -133,5 +139,10 @@
             <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>30.1.1-jre</version>
```

Comment:
This seems addressed by: https://github.com/confluentinc/kafka-connect-storage-common/pull/189
Let's exclude here.